compiler/src/compiler/ast.rs

```diff
@@ -125,6 +129,7 @@ pub enum AstError {
     StructWithNonStructField,
     StructWithoutClosingBrace,
     TextWithoutClosingQuote,
+    TextPlaceholderWithoutClosingCurlyBraces,
```

Comment:
```suggestion
    TextInterpolationWithoutClosingCurlyBraces,
```

and related references